### PR TITLE
Added gif in README.md

### PR DESCRIPTION
**Pull request by AK-007**
Issue #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +9,6 @@ rTunes is a web application developed on [Angular2](https://angular.io/) that us
 Clone the Repository followed by:
 
 - `npm install`
-- `npm start`
\ No newline at end of file
+- `npm start`
+
+![rtune](https://user-images.githubusercontent.com/26346816/34041794-b2440e98-e1bf-11e7-9689-36b92bba0efe.gif)
```
Comment (jajodiaraghav) [inline README.md]: Change rtune to `rTunes`
Don't forget to use `git commit --amend --no-edit`

**Comment by jajodiaraghav** [comment]
You have created your PR from master branch which is wrong. Always create a new branch for every PR

**Comment by AK-007** [comment]
I changed the README.md file in local repository.Now how should I commit the change through git?

**Comment by jajodiaraghav** [comment]
Steps:
1. `git add .`
2. `git commit --amend --no-edit`
3. `git push origin master -f`

**Comment by jajodiaraghav** [comment]
@AK-007 This cannot be merged if `package-lock.json` file remains in the diff. Remove the file and amend again so that this can be merged 👍

**Comment by AK-007** [comment]
I used 
1. ```git checkout "hash" -- package-lock.json```
2. ```git commit --amend --no-edit```
3. ```git push origin master -f ```
but still the file is in the commit.Can you help me out?

**Comment by jajodiaraghav** [comment]
Steps:
1. `git reset --soft head~1`
2. `git checkout -- package-lock.json`
3. `git checkout -- package.json`
4. `git status` [Should only show Readme.md]
5. `git add .`
6. `git commit`
7. `git push origin master -f`

**Comment by jajodiaraghav** [comment]
Merged 🚀 

Some suggestions:
1. Learn how to amend changes to last commit instead of writing multiple commits for just one small fix.
2. Go through https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
3. Always create new branch for new PR

**Pull request merged**